downstream/modules/aap-hardening/ref-interactive-session-timeout.adoc

```diff
@@ -0,0 +1,24 @@
+[id="ref-interactive-session-timeout"]
+
+=  Interactive session timeout
+
+A compliance profile might require that an interactive session timeout be enforced. 
+For example, the DISA STIG requires that all users be automatically logged out after 15 minutes of inactivity. 
+The installation process often requires an hour or more to complete, and this control can stop the installation process and log out the user before installation is complete. 
+The same also applies to day two operations such as backup and restore, which in production environments often take longer than the recommended interactive session timeout. 
+During these operations, the interactive session timeout should be increased to ensure the operation is successful.
+
+There are multiple ways in which this control may be enforced, including shell timeout variables, setting the idle session timeout for `systemd-logind`, or setting SSH connection timeouts, and different compliance profiles can use one or more of these methods. 
```

Comment:
Suggest changing "may" to "can" or "might", per the IBM style guide, because this is not s situation where permissions are indicated.